external/dynamo/generalized/router.py

```
@@ -30,15 +18,37 @@
+
+        def __init__(self, scores: dict[int, float] | None = None):
+            self.scores = scores if scores is not None else {}
+
+    class KvIndexer:
+        """Fallback: KV cache indexer for finding overlap between requests and workers."""
+
+        def __init__(self, engine: Any, block_size: int):
+            self.engine = engine
+            self.block_size = block_size
+
+        async def find_matches_for_request(self, tokens: list[int], min_overlap: int) -> OverlapScores:
```

Ruff (0.14.10)

Unused method argument: `tokens`

(ARG002)